src/targets/services/attributesHelpers.spec.js

```diff
@@ -0,0 +1,70 @@
+import { findNewAttributes, mergeAttributes } from './attributesHelpers'
+
+describe('attributesHelpers', () => {
+  const newIdentity = {
+    contact: { name: 'John', age: 32 }
+  }
+  let currentMyselfContactMock
+
+  beforeEach(() => {
+    jest.clearAllMocks()
+    currentMyselfContactMock = {
+      id: 1
+    }
+  })
+
+  const contactCollection = {}
+  contactCollection.update = jest.fn()
+  const mockLog = jest.spyOn(console, 'log')
+
+  describe('when there are new attributes', () => {
+    it('should be found', () => {
+      const newAttributesFound = findNewAttributes(
+        newIdentity.contact,
+        currentMyselfContactMock
+      )
+      expect(Object.keys(newAttributesFound).length).toBe(2)
+      expect(newAttributesFound).toEqual({ name: 'John', age: 32 })
+    })
+  })
```

Comment:
Not sure if this test is necessary since findNewAttributes is a detail of implementation of mergeAttributes ? And also, this function will be tested through mergeAttributes.